hardinfo.1

```diff
@@ -0,0 +1,57 @@
+.\" Manpage for hardinfo.
+.\" https://github.com/lpereira/hardinfo/.
+.TH HARDINFO "1" "April 2017" "0.5" "User Commands"
```

Comment:
done! that version and date was in the firs time i do the manpage, in the future the manpage must be generated dinamically maybe help2man, i-m working ont htat mechanish by my own but depends of some coordination beetbeen commits here